pkg/iac/deserializers.go

```diff
@@ -58,8 +58,8 @@ func Deserializers() []deserializer.CTYDeserializer {
 		awsdeserializer.NewCloudfrontDistributionDeserializer(),
 		awsdeserializer.NewKMSKeyDeserializer(),
 		awsdeserializer.NewKMSAliasDeserializer(),
-
 		ghdeserializer.NewGithubRepositoryDeserializer(),
 		ghdeserializer.NewGithubTeamDeserializer(),
+		awsdeserializer.NewLambdaEventSourceMappingDeserializer(),
```

Comment:
I would write it just below other aws deserializers

Reply:
ok but the list is already unordered ^^